### PR TITLE
Reorganization

### LINKED ISSUE #13: CSS style rendering right after pages are loaded.
Presently, the HTML text is rendered before the CSS style is loaded in causing a very brief moment where the unformatted HTML page is visible. Should hopefully be fixable by loading the stylesheet in the head instead of the body.

### LINKED ISSUE #14: Folder structuring
The current folder structure is a little awkward given the number of HTML files and other assets. I believe the assets folder should be removed and an HTML folder created. For the time being, I propose the following layout:

```
/
├─ html/
├─ css/
├─ cursors/
├─ fonts/
├─ js/
├─ media/
├─ .gitignore
├─ index.html
├─ README.md
```

Perhaps instead of `cursors/`, `fonts/`, and `media/` folders, these would be sorted into a more appropriate `assets/` folder.

This restructuring would also require the HTML files to be changed, updating links to the new paths in the folder structure.

### LINKED ISSUE #15: Remove old/unused files
There is a significant amount of old and unused files that we should remove to keep organized.

### PR DESCRIPTION
**Pull request by marcus-k**
This is the overhaul of the website folder structure to make it more organized and standard, which resolves #14.

Further, this fixes #13 by moving the CSS and JS loading from the body to the head.

Lastly, this closes #15 removing the old files everywhere and storing them in Google drive instead.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -1,223 +1,227 @@
...
+
+function copyToClipboard(text) {
+    var $temp = $("<input>");
+    $("body").append($temp);
+    $temp.val(text).select();
+    document.execCommand("copy");
+    $temp.remove();
+}
+
+console.log()
```
Comment (awsmit07) [inline js/script.js:180]: Why do we have a random console.log here?

**Comment by marcus-k** [comment]
Fixed merge conflicts.

**Review by awsmit07 — CHANGES_REQUESTED** [review]

File: js/script.js (hunk 1 above)
Comment (marcus-k) [reply js/script.js:180]: I noticed a bunch of depreciated code, old comments, and useless sections too that I think we can probably address in a future PR.

Comment (marcus-k) [reply js/script.js:180]: I'll remove this one for the time being though.

**Review by awsmit07 — APPROVED** [review]

**Pull request merged**